staging/src/k8s.io/apiserver/pkg/server/filters/goaway_test.go

```diff
@@ -438,10 +436,8 @@ func TestGOAWAYHTTP1Requests(t *testing.T) {
 }
 
 // TestGOAWAYConcurrency tests GOAWAY frame will not affect concurrency requests in a single http client instance.
-// Known issues in history: https://github.com/kubernetes/kubernetes/issues/91131.
+// Known issues in history: https://github.com/kubernetes/kubernetes/issues/91131, https://github.com/golang/go/issues/41234.
```

Comment:
Given that the flake is fixed, we don't need to record history in the comment anymore. Can you remove this line?

Reply:
Already done, thanks